curriculum/challenges/english/blocks/lab-feature-selection/68ce56d54d3cad1149a1c4cf.md

```diff
@@ -0,0 +1,289 @@
+assert.isNotEmpty(labels);
+
+labels.forEach(label => assert.isNotEmpty(label.innerText.trim()));
+```
+
+Inside each label element there should be one `input` element of type `checkbox`.
+
+```js
+const labels = document.querySelectorAll('.feature-card-container label');
+assert.isNotEmpty(labels);
+
+labels.forEach(label => assert.exists(label.querySelector('input[type="checkbox"]')));
+```
+
+You should have an `input[type="checkbox"]` selector.
+
+```js
+assert.exists(new __helpers.CSSHelp(document).getStyle('.feature-card input[type="checkbox"]'));
+```
+
+Your checkboxes should be set to `appearance: none`.
+
+```js
+assert.equal(new __helpers.CSSHelp(document).getStyle('.feature-card input[type="checkbox"]').getPropVal('appearance'), 'none');
+```
+
+Your checkboxes should have a border width of your choosing.
+
+```js
+assert.isNotEmpty(new __helpers.CSSHelp(document).getStyle('.feature-card input[type="checkbox"]').getPropVal('border-width'));
+```
+
+Your checkboxes should have a border color of your choosing.
+
+```js
+assert.isNotEmpty(new __helpers.CSSHelp(document).getStyle('.feature-card input[type="checkbox"]').getPropVal('border-color'));
+```
+
+Your checkboxes should have a border style of your choosing.
+
+```js
+assert.isNotEmpty(new __helpers.CSSHelp(document).getStyle('.feature-card input[type="checkbox"]').getPropVal('border-style'));
+```
+
+When the checkboxes are selected, there should be a checkmark `✓` in the center of the checkbox.
+
+```js
+const checkboxes = document.querySelectorAll('.feature-card input[type="checkbox"]');
+assert.isNotEmpty(checkboxes);
+
+checkboxes.forEach((checkbox) => { 
+    checkbox.checked = false;
+
+    checkbox.checked = true;
+    const after = window.getComputedStyle(checkbox, ':after');
+    assert.equal(after.content, '"✓"');
+});
+```
+
+When the checkbox is checked, the background color of the checkbox should change to a color of your choosing.
+
+```js
+const checkboxes = document.querySelectorAll('.feature-card input[type="checkbox"]');
+assert.isNotEmpty(checkboxes);
+
+checkboxes.forEach((checkbox) => { 
+    checkbox.checked = false;
+    const uncheckedBG = window.getComputedStyle(checkbox).backgroundColor;
+    checkbox.checked = true;
+    const checkedBG = window.getComputedStyle(checkbox).backgroundColor;
+
+    assert.notEqual(checkedBG, uncheckedBG);
+});
+```
+
+When the checkbox is checked, the border color of the checkbox should change to a color of your choosing.
+
+```js
+const checkboxes = document.querySelectorAll('.feature-card input[type="checkbox"]');
+assert.isNotEmpty(checkboxes);
+
+checkboxes.forEach((checkbox) => { 
+    checkbox.checked = false;
+    const uncheckedBC = window.getComputedStyle(checkbox).borderColor;
+    checkbox.checked = true;
+    const checkedBC = window.getComputedStyle(checkbox).borderColor;
+
+    assert.notEqual(checkedBC, uncheckedBC);
+});
+```
+
+# --seed--
+
+## --seed-contents--
+
+```html
```

Comment:
Will have time to review lab later on. But noticed this doesn't have the starting HTML boilerplate.
It looks like the other labs have this boilerplate like here 

https://www.freecodecamp.org/learn/full-stack-developer/lab-contact-form/design-a-contact-form